### ISSUE TITLE
Redesign dataset search

### ISSUE THREAD

**Issue by mdoering** [CatalogueOfLife/checklistbank#1116]
I am missing facet counts for most dataset search parameters (origin, type, license, publisher, hasGbifKey, ...) and I think it would be great to also include always the latest dataset import metrics into the search. This would allow us to search for datasets with certain entities (hasVernacular, hasMedia, etc), the dataset format, or even counts if we wanted (numMedia > 10.000).

Larger numbers of facets would probably require the search to be driven by Elastic, which would be a larger refactoring on the backend. First think through here from the user perspective what a better dataset search should be able to offer.

**Comment by olafbanki** [comment]
Agreed Markus, we should get the collated feedback from the ChecklistBank workshops first to understand what at first sight people are looking for in ChecklistBank. Then see what the main and common user stories are for search in ChecklistBank, and the prioritise these. Refacturing Elastic might be a substantial piece of work. Let us chat about this; also how it fits in with other planned work.

**Comment by mdoering** [comment]
@olafbanki @chantalhuijbers any new insights into this or other requests from the workshops so far?
